.gitignore

```diff
@@ -27,3 +27,5 @@ fp-info-cache
 # Auto-generated directories by VS Code
 .vscode
 .pio
+images/.DS_Store
```

Comment:
I believe this is covered by the next line. But its fine to leave as-is in case I'm wrong :)